src/modules/grants/components/GrantsHeader.tsx

```diff
@@ -0,0 +1,191 @@
+import { Box, Button, HStack, Image, Skeleton, VStack } from '@chakra-ui/react'
+import { useTranslation } from 'react-i18next'
+import { PiCoins, PiTrophy } from 'react-icons/pi'
+import { useNavigate } from 'react-router'
+import { Link } from 'react-router-dom'
+
+import { Body } from '@/shared/components/typography'
+import { getPath, GrantsFAQUrl, GrantsPageBannerNoiseGifUrl } from '@/shared/constants'
+import { useGrantStatisticsQuery } from '@/types'
+import { getShortAmountLabel, useMobileMode } from '@/utils'
+
+// TODO: Change to geyser before pushing to production
+const GEYSER_PROJECT_NAME = 'geyser2'
```

Comment:
You can use, __production__, __staging__ check for having a different variables based on envs.